Formula/jank-git.rb

```diff
@@ -1,7 +1,7 @@
-class Jank < Formula
+class JankGit < Formula
   desc "The native Clojure dialect hosted on LLVM with seamless C++ interop."
   homepage "https://jank-lang.org"
-  url "https://github.com/jank-lang/jank.git", branch: "main"
+  url "https://github.com/oscarvarto/jank.git", branch: "main"
```

Comment:
This is temporary, and has my changes in https://github.com/jank-lang/jank/pull/585